packages/server/lib/browsers/cri-client.ts

```
@@ -144,7 +143,11 @@ const maybeDebugCdpMessages = (cri: CDPClient) => {
     cri._ws.send = (data, callback) => {
       debugVerboseSend('sending CDP command %o', JSON.parse(data))
 
-      return send.call(cri._ws, data, callback)
+      try {
```

Is this going to end up masking problems that we may end up having during development? I think I've run into scenarios where I've coded something wrong and it'll throw an error. I worry that might be hard to track down if we just swallow those errors like this.

---

Changed to re-throw the error on L#150 in https://github.com/cypress-io/cypress/pull/29676/commits/706a1e3d35e2b14e81db03db3de6f78eacc93f8a